Gtk-4.0.gir

```diff
@@ -82849,6 +82899,11 @@ gtk_string_list_take (self, g_strdup_print ("%d dollars", lots));
           </parameter>
         </parameters>
       </method>
+      <property name="strings" readable="0" writable="1" construct-only="1" transfer-ownership="none">
```

Comment:
@bilelmoussaoui missing version :cry:

Reply:
already submitted a PR :) see the comment above